client/my-sites/importer/section-import.jsx

```diff
@@ -183,7 +182,7 @@ class SectionImport extends Component {
 				target="_blank"
 				rel="noopener noreferrer"
 			>
-				{ this.props.translate( 'Other importers' ) }
+				{ this.props.translate( 'Another service' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **4** times:
`translate( 'Add another service' )` *ES Score: 8*

Reply:
I'm not sure if "Another service" is descriptive enough for this link. 

@obi2020 Do you have any suggestions for copy here? 

![Screen Shot 2019-10-22 at 4 35 39 PM](https://user-images.githubusercontent.com/181733/67343393-0383ba80-f4ea-11e9-93ee-c83fbe400939.png)

Reply:
@lcollette Does this go to a support page?

Reply:
@obi2020 It goes to a /wp-admin Importers screen that looks like this:
![screencapture-calobeetesting05photo-wordpress-wp-admin-import-php-2019-10-23-11_48_56](https://user-images.githubusercontent.com/2124984/67411151-29ad6700-f58b-11e9-8075-1953ea6bcf5e.png)

Reply:
Thanks! 

What about something like "See full list" or "Choose from full list" ?

I actually don't mind the current "Another service" except that on the destination page we call them "systems" and not "services"